cwms-data-api/src/main/java/cwms/cda/api/EmbankmentController.java

```diff
@@ -79,7 +79,7 @@ private Timer.Context markAndTime(String subject) {
             },
             responses = {
                     @OpenApiResponse(status = STATUS_200, content = {
-                            @OpenApiContent(isArray = true, type = Formats.JSON, from = Embankment.class)
```

Comment:
I think you want to leave the Formats.JSON here. That will allow it to show up in the drop down properly.

Reply:
Good point.  When there are aliases we should probably put the more general in the combo box.  We could have both in there.

Reply:
added